addons/hr_expense_stripe/models/res_company.py

```diff
@@ -59,56 +69,72 @@ def _connect_to_stripe(self):
-                    'enabled_events[]': HANDLED_WEBHOOK_EVENTS,
-                    'api_version': API_VERSION,
-                },
-                method='POST',
-            )
-            self.stripe_webhook_secret = webhook.get('secret')
-            message = _("You Stripe Webhook was successfully set up!")
-            notification_type = 'info'
 
+        if self.stripe_webhook_secret:
+            raise UserError(_("A Webhook URL already exists for this company."))
+
+        self.stripe_webhook_secret = uuid.uuid4()
+
+    def action_create_stripe_account(self):
+        self.ensure_one()
+        if self.stripe_account_id:
+            raise UserError("User is already connected to stripe issuing.")
+        self._create_webhook_secret()
+
+        payload = {
+            # IAP Data
+            'webhook_url': self._get_stripe_webhook_url(),
+            'country': self.country_id.code,
+        }
+
+        response = make_request_stripe_proxy('api/stripe_issuing/1/accounts', payload, method='POST')
+        self.stripe_account_id = response['stripe_ident']
+
+        # Now that we have created the account, we redirect the user to Stripe to let him configure it 
+        payload = {
+            'account': self.stripe_account_id,
+            'refresh_url': self.get_base_url(),
+            'return_url': self.get_base_url(),
+        }
+        response = make_request_stripe_proxy('api/stripe_issuing/1/account_links', payload, method='POST')
         return {
-            'type': 'ir.actions.client',
-            'tag': 'display_notification',
-            'params': {
-                'message': message,
-                'sticky': False,
-                'type': notification_type,
-                'next': {'type': 'ir.actions.act_window_close'},  # Refresh the form to show the key
+                'type': 'ir.actions.act_url',
+                'url': response['onboarding_url'],
+                'target': 'self',
             }
+
+    def action_refresh_stripe_account(self):
+        self.ensure_one()
+
+        if not self.stripe_account_id:
+            raise ValidationError(_("You need to be connected to stripe in order to refresh your account."))
+
+        response = make_request_stripe_proxy(f'api/stripe_issuing/1/accounts/{self.stripe_account_id}', method='GET')
+        self.stripe_account_issuing_status = 'verified' if response['capabilities']['card_issuing'] == 'active' else 'restricted' 
+
+    def action_configure_stripe_account(self):
+        self.ensure_one()
+
+        if not self.stripe_account_id:
+            raise ValidationError(_("You need to be connected to stripe in order to configure your account."))
+
+        payload = {
+            'account': self.stripe_account_id,
+            'refresh_url': self.get_base_url(),
+            'return_url': self.get_base_url(),
+        }
+        response = make_request_stripe_proxy('api/stripe_issuing/1/account_links', payload, method='POST')
```

Comment:
```suggestion
        response = make_request_stripe_proxy('/account_links', payload, method='POST')
```
If it's always the same, why mentioning it every time and not put it in the method?